src/chapters/git/exercises.rst

```diff
@@ -0,0 +1,136 @@
+Exercises: Git More Practice
+============================
+
+Working with a Local Repository
+-------------------------------
+
+Use your terminal powers to move through these Git exercises.
+
+#. Launch Visual Studio Code and open the terminal pane.
+#. In your ``local_practice`` folder, make a new directory called
+   ``Git-Exercises``. Next, go to *File --> Open* and select this new folder.
+#. In the terminal, make sure that you are in the ``Git-Exercises`` directory.
+   Initialize it as a new repository with ``git init``.
+#. Add two files to the directory: ``index.html`` and ``style.css``. Use the
+   ``status/add/commit`` commands to commit the changes to the repository.
+#. Paste in this boilerplate HTML code into ``index.html``, then commit again.
+
+   .. sourcecode:: html
+      :linenos:
+
+      <!DOCTYPE html>
+      <html>
+         <head>
+            <meta charset="utf-8">
+            <meta name="viewport" content="width=device-width">
+            <title>Commit to Git!</title>
+            <link href="style.css" rel="stylesheet" type="text/css" />
+         </head>
+         <body>
+            
+         </body>
+      </html>
+
+#. Use ``git checkout -b`` to create a new branch called ``html-code``. While
+   in that branch, add at least 5 new elements to the HTML file. Save and
+   commit your changes.
+#. Return to ``main``. Use ``checkout`` again to create a third branch called
```

Comment:
As noted earlier, will the default branch be `main` if they create the repo with `git init` instead of cloning from GitHub?

Reply:
I added a sentence in step 3 about changing the default branch name to `main`.